test/integration/rails_gems_full_stack_test.rb

```diff
@@ -1,36 +0,0 @@
-# frozen_string_literal: true
```

Comment:
@danmayer Was running into trouble having both rails_gem_full_stack_test and rails_full_stack_test as rails can only be initialized once per test run.